src/main/scala/cromwell/binding/types/WdlObjectType.scala

```
@@ -1,16 +1,30 @@
 package cromwell.binding.types
 
 import cromwell.binding.Call
-import cromwell.binding.values.{WdlCallOutputsObject, WdlObject}
+import cromwell.binding.values.{WdlCallOutputsObject, WdlMap, WdlObject, WdlString}
+
+import scala.util.{Failure, Success}
 
 case object WdlObjectType extends WdlType {
   val toWdlString: String = "Object"
 
   override protected def coercion = {
     case o: WdlObject => o
+    case m: WdlMap if m.wdlType.keyType == WdlStringType && WdlStringType.isCoerceableFrom(m.wdlType.valueType) =>
+      val coercedMap = m.value map {
+        case (k,v) => k.asInstanceOf[WdlString].value -> WdlStringType.coerceRawValue(v).map(_.asInstanceOf[WdlString])
+      } collect {
+        case (k, Success(v)) => k -> v
+        case (_, v: Failure[_]) => throw new UnsupportedOperationException(s"Unable to convert value to String")
```

maybe propagate the exception and/or its message?